source/libraries/ContractDeployer.ts

```diff
@@ -28,7 +28,13 @@ export class ContractDeployer {
         this.contracts = new Contracts(compilerOutput);
     }
 
+    public async getBlockNumber(): Promise<number> {
+        return this.connector.ethjsQuery.getBlockByNumber('latest', false).then( (block) => block.number.toNumber());
+    }
+
     public async deploy(): Promise<void> {
+        console.log("Saving contract start block");
```

Comment:
Recommend removing this log line.  If it is desirable to know when `deploy` starts, then text should be changed to `Starting contract deployment.` or something (the start block thing is fairly minor relative to the rest of the process).